gateway/conf.d/jaeger.conf

```diff
@@ -0,0 +1,4 @@
+jaeger_reporter_local_agent_host_port {{ jaeger_addr | default: '127.0.0.1' }}:{{ jaeger_port | default: 6831 }};
```

Comment:
We should add `.liquid` extension to files that are processed by liquid so it is more obvious liquid is available there.